### ISSUE TITLE
Quarto doc divs and alt text

### ISSUE THREAD

**Issue by dmgatti** [smcclatchy/eqtl-mapping#7]
When I run validate_lesson(), I get errors about there being no "notes" div label and figures missing alt text in instructors/eqtl-mapping.qmd.

> sandpaper::validate_lesson()
── Validating Fenced Divs ──────────────────────────────────────────────────────────────────────────────────────────────────────────────
! There were errors in 9/109 fenced divs
◌ The Carpentries Workbench knows the following div types callout, objectives, questions, challenge, prereq, checklist, solution, hint, discussion, testimonial, keypoints, instructor, spoiler, tab, group-tab

instructors/eqtl-mapping.qmd:11 [unknown div] notes
instructors/eqtl-mapping.qmd:24 [unknown div] notes
instructors/eqtl-mapping.qmd:38 [unknown div] notes
instructors/eqtl-mapping.qmd:47 [unknown div] notes
instructors/eqtl-mapping.qmd:57 [unknown div] notes
instructors/eqtl-mapping.qmd:67 [unknown div] notes
instructors/eqtl-mapping.qmd:78 [unknown div] notes
instructors/eqtl-mapping.qmd:89 [unknown div] notes
instructors/eqtl-mapping.qmd:109 [unknown div] notes
── Validating Internal Links and Images ────────────────────────────────────────────────────────────────────────────────────────────────
! There were errors in 8/126 images
◌ Images need alt-text <https://webaim.org/techniques/hypertext/link_text#alt_link>

instructors/eqtl-mapping.qmd:9 [image missing alt-text]: ../episodes/fig/cystic-fibrosis.png
instructors/eqtl-mapping.qmd:22 [image missing alt-text]: ../episodes/fig/gwas-catalog.png
instructors/eqtl-mapping.qmd:32 [image missing alt-text]: ../episodes/fig/geneticstype2-Image001.jpg
instructors/eqtl-mapping.qmd:53 [image missing alt-text]: ../episodes/fig/trans-noncoding-SNP.png
instructors/eqtl-mapping.qmd:63 [image missing alt-text]: ../episodes/fig/healthy-vs-T2D.png
instructors/eqtl-mapping.qmd:74 [image missing alt-text]: ../episodes/fig/pancreatic-islet.png
instructors/eqtl-mapping.qmd:87 [image missing alt-text]: ../episodes/fig/gtt_auc_example.png
instructors/eqtl-mapping.qmd:107 [image missing alt-text]: ../episodes/fig/insulin-auc-lod-plot.png
